tests/test_optim.py

```
@@ -599,3 +599,25 @@ def test_minuit_all_fixed_params():
+    # Regression test: _internal_postprocess must not raise when a fit result
+    # carries correlations but no uncertainties (num_fixed_pars was previously
+    # only defined inside the uncertainties branch).
+    pyhf.set_backend("numpy")
+
+    fitresult = OptimizeResult(
+        x=np.asarray([1.0, 2.0]),
+        fun=0.0,
+        corr=np.asarray([[1.0, 0.5], [0.5, 1.0]]),
+    )
+
+    def stitch_pars(pars, stitch_with=None):  # noqa: ARG001
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `stitch_pars`

(ANN202)